packages/utilities-deprecated/projects/utilities/CHANGELOG.md

```diff
@@ -9,6 +9,706 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+```
+
+- `fontWeight` is provided the same way, only the import has changed.
+
+```diff
+- import { fontWeight } from '@porsche-design-system/utilities';
++ import { fontWeight } from '@porsche-design-system/components-{js|angular|react|vue}/styles';
+```
+
+**Spacings**
+
+With the upcoming v3 release, we will be introducing fluid spacers alongside our static spacers. These spacers will
+adjust in size based on a min and max value, providing a fluid response to changes in browser width.
+
+```diff
++ import { spacing, spacingFluid } from '@porsche-design-system/components-{js|angular|react|vue}/styles';
+```
+
+Additionally, we've switched from using _rem_ to _px_ for our spacers to ensure they don't unnecessarily expand when the
+root font size is adjusted.
+
+```diff
+- import { layout } from '@porsche-design-system/utilities';
++ import { spacingStatic } from '@porsche-design-system/components-{js|angular|react|vue}/styles';
+import styled from 'styled-components';
+
+const Component1 = styled.div({
+- margin: layout.medium
++ margin: spacingStatic.medium
+});
+```
+
+To get an overview ot the new spacings have a look [here](https://designsystem.porsche.com/latest/styles/spacing).
+
+##### SCSS
+
+```diff
+- @import '~@porsche-design-system/utilities/scss';
++ @import '~@porsche-design-system/components-{js|angular|react|vue}/styles/scss';
+```
+
+**Mixins**
+
+- `p-focus` is now `pds-focus`.
+
+```diff
+- @import '~@porsche-design-system/utilities/scss';
++ @import '~@porsche-design-system/components-{js|angular|react|vue}/styles/scss';
+
+a {
+-  @include p-focus;
++  @include pds-focus;
+  }
+
+button {
+-  @include p-focus($p-color-state-focus, 1px, '::before');
++  @include pds-focus('small', 'small', 'light');
+  }
+```
+
+- `breakpoint` `xxs` is now `base`.
+- instead of `p-media-query` we provide now `pds-media-query-min`, `pds-media-query-max` and `pds-media-query-min-max`.
+  Furthermore, the mixins only accept the predefined Porsche Design System breakpoints from `base` to `xxl` and no
+  custom breakpoints anymore.
+
+```diff
+- @import '~@porsche-design-system/utilities/scss';
++ @import '~@porsche-design-system/components-{js|angular|react|vue}/styles/scss';
+
+div {
+  color: inherit;
+
+   // up to predefined breakpoint xs apply color black
++  @include pds-media-query-max('xs')
+    color: black;
+  }
+
+  // from predefined breakpoint xs to m apply color aqua
+-  @include p-media-query('xs', 'm') {
++  @include pds-media-query-min-max('xs', 'm')
+    color: aqua;
+  }
+
+  // from predefined breakpoint m apply color deeppink
+-  @include p-media-query('m') {
++  @include pds-media-query-min('m')
+    color: deeppink;
+  }
+}
+```
+
+- `p-title-large` mixin is now renamed to `pds-display-large` and in addition we provide `pds-display-medium`. The size
+  of the new `display` styles are changing fluid depending on the viewport width instead of fixed predefined sizes on
+  specific breakpoints, and they are italic now.
```

Comment:
see comment above. Don't make detailed descriptions (they can be easily out of sync)